modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/twostep/msg/GridH2QueryRequest.java

```diff
@@ -693,6 +714,11 @@ public static Boolean isDataPageScanEnabled(int flags) {
 
                 writer.incrementState();
 
+            case 15:
```

Comment:
Looks like it was added by hands.
Why can't this be generated with MessageGenerator like GridH2DmlRequest?

Reply:
Because GridH2QueryRequest is annotated with @IgniteCodeGeneratingFail